pkg/multicluster/local/local_cluster_manager.go

```diff
@@ -106,9 +106,9 @@ func (cm *localClusterManager) DeleteBlueprint(cluster string, namespace string,
 }
 
 // NewClusterManager creates an instance of ClusterManager for a local cluster configuration
-func NewClusterManager(client client.Client, namespace string) (multicluster.ClusterManager, error) {
+func NewClusterManager(clnt client.Client, namespace string) (multicluster.ClusterManager, error) {
```

Comment:
Use cl for client as we do in some other files

Reply:
I'll change it